examples/python/Misc/MultiDevice/multi_device_frame_sync.py

```
@@ -227,21 +295,33 @@ def interruptHandler(sig, frame):
+                inputQueues[f"slave_{deviceName}_{socketName}"].send(camOutputQueue.get())
+            else:
+                time.sleep(0.001)
+
+    threads = {}
 
-        for k, v in slaveQueues.items():
-            for k2, v2 in v.items():
-                while v2.has():
-                    inputQueues[f"slave_{k}_{k2}"].send(v2.get())
+    for deviceName, sockets in slaveQueues.items():
+        for socketName, camOutputQueue in sockets.items():
```

Ruff (0.15.6)

[warning] Loop control variable `camOutputQueue` not used within loop body

Rename unused `camOutputQueue` to `_camOutputQueue`

(B007)